dali/python/nvidia/dali/external_source.py

```diff
@@ -131,10 +136,10 @@ def feed(self):
 
 
 class _ExternalSourceGroup(object):
-    def __init__(
-            self, callback, source_desc, is_multioutput, instances=[], *,
-            cuda_stream=None, use_copy_kernel=None, batch=True, parallel=False,
-            prefetch_queue_depth=None, batch_info=None):
+
+    def __init__(self, callback, source_desc, is_multioutput, instances=[], *, cuda_stream=None,
+                 use_copy_kernel=None, batch=True, parallel=False, prefetch_queue_depth=None,
+                 batch_info=None):
```

Comment:
The line break at `*` was intentional - to make it clear where the keyword-only arguments start.

Reply:
Done